src/components/screens/NewsItem.js

```diff
@@ -113,7 +124,19 @@ export const NewsItem = ({ data }) => {
             !!mediaContent.sourceUrl.url &&
             section.push(
               <WrapperHorizontal key={`${index}-${contentBlock.id}-mediaContent${mediaContent.id}`}>
-                <HtmlView html={trimNewLines(mediaContent.sourceUrl.url)} />
+                <WebView
+                  source={{ html: trimNewLines(mediaContent.sourceUrl.url) }}
```

Comment:
what is this helper doing? I can't find the commit for it.

Reply:
you mean the helper `trimNewLines`?  it is defined in [/src/helpers/htmlViewHelper.js](https://github.com/ikuseiGmbH/smart-village-app-app/blob/master/src/helpers/htmlViewHelper.js). it is used for removing unnecessary and unwanted new lines in a text, that we want to render as html.